git/git.go

```diff
@@ -218,10 +218,11 @@ func (provider *repositoryProvider) dumpFile(repository *git.Repository, name st
 		return nil
 	}
 
+    fileName := filepath.Base(filePath)
```

Comment:
this is a very small nit but the indentation seems off

Reply:
It's not